src/main/webapp/app/components/account/ActivateTrialFinish.tsx

```diff
@@ -53,6 +57,7 @@ export default class ActivateTrialFinish extends React.Component<{
           key: this.activateKey,
           readAndAgreeWithTheTerms: true,
         },
+        newAccountCreation: this.newAccountCreation,
       })
       .then(
         user => {
```

Comment:
Ah seems like we already implemented the redirecting. Let's shorten the time out .